components/bl-collapsiblePanel-component/component.json

```diff
@@ -0,0 +1,120 @@
+{
+  "id": "c_f6882cbc4c8892c8cc465b8d23d45e3a",
+  "name": "CollapsiblePanel",
+  "description": "The CollapsiblePanel component displays summary information and upon selecting the panel, it expands to display the full expansion panel.",
+  "showInToolbox": true,
+  "faIcon": "window-maximize",
+  "mainJS": "dist/index.js",
+  "type": "custom",
+  "category": "Custom Components",
+  "properties": [
+    {
+      "type": "text",
+      "name": "width",
+      "label": "Width",
+      "settingTooltip": "Width of the panel",
+      "showInSettings": true,
+      "hasLogicHandler": false,
+      "handlerId": "widthLogic"
+    },
+    {
+      "type": "text",
+      "name": "titleDefault",
+      "label": "Title",
+      "showInSettings": true,
+      "hasLogicHandler": true,
+      "handlerId": "titleDefaultLogic",
+      "dataBinding": true,
+      "settingTooltip": "Default title of the panel",
+      "handlerLabel": "Title Logic",
+      "handlerDescription": "This is a handler for the logic to determine the title of the panel.",
+      "defaultValue": "Panel Title"
+    },
+    {
+      "type": "text",
+      "name": "titleOpen",
```

Comment:
Let's remove this property. I think the user could define title content from one place, the second handler could just confuse user.

Reply:
I think it's better to leave. Very useful, for example, in the case "more information / less information".